plugins/available/base.plugin.bash

```diff
@@ -88,16 +88,15 @@ function pmdown ()
       echo "You don't have a markdown command installed!"
     fi
 }
-
+#if using mkcd many dirs, it will make all dirs and cd into the lastest one.
 function mkcd ()
 {
     about 'make a directory and cd into it'
     param 'path to create'
```

Comment:
Please update the description of the `param` to indicate that one or more directories can be provided as parameters.